std/json.d

```diff
@@ -1706,12 +1706,14 @@ pure nothrow @safe unittest // issue 15884
 
 @safe unittest // issue 15885
 {
+    enum bool realInDoublePrecision = real.mant_dig == double.mant_dig;
+
     static bool test(const double num0)
     {
         import std.math : feqrel;
         const json0 = JSONValue(num0);
         const num1 = to!double(toJSON(json0));
-        version(Win32)
+        static if (realInDoublePrecision)
```

Comment:
> Is there a list online somewhere of which platforms have is(double == real)?

It would be easier to list those that *don't*.  x86 (80-bit) and PPC (64-bit pair) for sure, I think IA64 (128-bit) is in that list too.  Everyone else is a platform where real == double.